devtools-html/debugger.html
# Symbol Search Modal

Associated Issue: #2587 

### Summary of Changes

* rework the symbol search UI into a modal separate from file search
* simplify active search state (which is shown, project, file or symbol)

### Test Plan

* Open symbol modal (<kbd>ctrl</kbd>/<kbd>cmd</kbd>+<kbd>shift</kbd>+<kbd>o</kbd>
* Use `@` + query for functions
* Use `#` + query for variables
* Confirm that results match and traversing results travels properly.
